pkg/query/testdata/promql/thanos/aggregators.test

```diff
@@ -0,0 +1,218 @@
+store {} 0 30m
```

Comment:
It's 1:1 copied from Prometheus, except beginning where we load multiple stores.